registry.json

```diff
@@ -4425,5 +4425,19 @@
    "telegram": "lunarpool_io",
    "operatorName": "Dusan",
    "github": "lunarpool"
+  },
+  {
+   "ticker": "ARA",
+   "poolId": "aed2477a1b56db48312eb9dc98803b482e33e51fd37d23e6817051e6",
+   "social": {
+     "website": "https://ara-stakepool.com/",
+     "twitter": "arastakepool",
+     "facebook": "",
+     "instagram": "",
+     "youtube": ""
+   },
+   "telegram": "arastakepool",
```

Comment:
Please can you change this to be the TG id of an individual and not a channel or group?

Reply:
Done :)